File: corpus/typescript/fixture_21.ts
type CartItem21 = { price: number; qty?: number };
export function calculateDiscount21(cartItems: CartItem21[], taxRate: number): number {
  const totalAmount = cartItems.reduce((sum, item) => sum + item.price * (item.qty ?? 1), 0);
  return totalAmount > 210 ? totalAmount * (1 + taxRate) : totalAmount;
}

export class PriceBook21 {
  lookupPrice(sku: string): number { return sku.length + 21; }
}

File: corpus/python/fixture_21.py
def calculate_discount_21(cart_items, tax_rate):
    total_amount = sum(item["price"] * item.get("qty", 1) for item in cart_items)
    if total_amount > 210:
        return total_amount * (1 + tax_rate)
    return total_amount

class PriceBook21:
    def lookup_price(self, sku):
        return len(sku) + 21
